plugin.go

```diff
@@ -103,9 +102,14 @@ func installCaCert(cacert string) *exec.Cmd {
 	)
 }
 
-func exportSecrets(secrets map[string]string) {
-	for k, v := range secrets {
-		os.Setenv(fmt.Sprintf("%s", k), fmt.Sprintf("%s", os.Getenv(v)))
+func CopyTfEnv() {
+	tfVar := regexp.MustCompile(`^TF_VAR_.*$`)
+	for _, e := range os.Environ() {
+		pair := strings.Split(e, "=")
+		if tfVar.MatchString(pair[0]) {
+			name := strings.Split(pair[0], "TF_VAR_")
+			os.Setenv(fmt.Sprintf("TF_VAR_%s", strings.ToLower(name[1])), pair[1])
```

Comment:
It's a bit of a slippery slope to lowercase everything. A name of a variable in TF does not need to be all lowercase. I could have something like `TF_VAR_myListOfCIDRs` which won't work/get fixed with this change. 

I think the problem is on the drone side, where it's uppercasing everything, maybe unnecessarily, if there are use cases what would not want it to happen. I feel like fixing it in the plugin is not a right place to do so.